joomla/joomla-cms
# [4.0] Clean up the system templates (assets)

Pull Request for Issue # . Part of the Child template effort

### Summary of Changes
- Move all the scss files to `/media/system/scss`
- Adjust all the stylesheet includes


### Testing Instructions
- You need to run `npm install`
- for the front end you need to add `$template->template = 'system';` after line 546 in the file https://github.com/joomla/joomla-cms/blob/7935d489c876155aa3ba1ccfa911a2ad6dd5a79e/libraries/src/Application/SiteApplication.php#L591 then visit your local site (eg localhost/)
- for the back end you need to add `$template->template = 'system';` after line 270 in the file https://github.com/joomla/joomla-cms/blob/7935d489c876155aa3ba1ccfa911a2ad6dd5a79e/libraries/src/Application/AdministratorApplication.php#L269 then visit your administrator part of your local site (eg localhost/administrator)
- For both cases check the assets loaded in the head of the document

![Screenshot 2020-12-25 at 13 06 00](https://user-images.githubusercontent.com/3889375/103134823-50e25580-46b4-11eb-8747-4e181392040d.png)
![Screenshot 2020-12-25 at 13 09 49](https://user-images.githubusercontent.com/3889375/103134825-52ac1900-46b4-11eb-83a3-62448eb2e65e.png)

### Actual result BEFORE applying this Pull Request



### Expected result AFTER applying this Pull Request
It works exactly as before, the static assets were just moved into the `/media` folder

### Documentation Changes Required
IDK the system templates are supposed to act as fallback if something goes horrible wrong with the current active template (but the fallback is not pretty or functional...)
